src/summary/history/conditions-card.component.tsx

```diff
@@ -35,7 +37,7 @@ export default function ConditionsCard(props: ConditionsCardProps) {
   }, [patient]);
 
   return (
-    <SummaryCard name="Conditions" match={props.match}>
+    <SummaryCard name={t("conditions", "Conditions")} match={props.match}>
```

Comment:
Per @joeldenning's help, you can use this function instead of the <Trans> element when translating a string being passed as a property. 

Note, this probable doesn't belong in this PR but was a test case and such a small amount of code that I'm including it here.